torch_xla/csrc/tensor_ops.cpp

```diff
@@ -127,16 +130,17 @@ XLATensor SmoothL1Loss(const XLATensor& input, const XLATensor& target,
 
 XLATensor SmoothL1LossBackward(const XLATensor& grad_output,
                                const XLATensor& input, const XLATensor& target,
-                               ReductionMode reduction) {
+                               ReductionMode reduction, double beta) {
   auto broadcasted_inputs = XLATensor::broadcast_tensors({input, target});
```

Comment:
Might be worth using a IR scope pusher here and in the backward:

https://github.com/pytorch/xla/blob/14d617a2ec11f7e579aa7eb39d5331331e2940c3/torch_xla/csrc/ops/ops.cpp#L185